scripts/deploy_to_prod.py

```diff
@@ -0,0 +1,11 @@
+import os
+import json
+
+prod_clients_json = os.environ['PROD_CLIENTS']
+
+prod_clients = json.loads(prod_clients_json)
+
+for client in prod_clients["prod_clients"]:
+    # We just print a fake list of hostnames for now
+    print(client["hostname"])
```

Comment:
it would be nice here to reuse the script that we use in `deploy_to_dev` and `deploy_to_uat` jobs. Also, you should try to connect on an actual server (via ssh) and run a command on this server (for example `echo $HOSTNAME`)

Reply:
We want to use the script that @ayyazdaniaryan is currently writing for prod deployment

Reply:
yeah, I know. I meant use the `scripts/echo.sh` to prove we can do it but I don't think it worth it since in the end it will depend on how the deploy script is done. 
But you could definitely do the ssh part (echo $HOSTNAME) on all servers defined in `prod_clients`

Reply:
Allright sure :)